restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -173,24 +209,26 @@ public void deleteThenInsertBatch() throws UnauthorizedException {
 
     assertThat(generatedQueries.get(0).queryString())
         .isEqualTo(
-            "DELETE FROM \"keyspace\".\"table\" USING TIMESTAMP ? WHERE key = :key AND p0 = :p0 AND p1 = :p1 AND p2 = :p2");
+            "DELETE FROM \"keyspace\".\"table\" USING TIMESTAMP ? WHERE key = ? AND p0 = ? AND p1 = ? AND p2 = ?");
```

Comment:
bind parameters(`:smth`) are more explicit than wildcard markers(`?`). Why we are changing this?

Reply:
because the way CQL text is generated has changed. We do not directly control the CQL, it is produced by the query build and it now uses `?` for parameters.